<?xml version="1.0" encoding="UTF-8"?>
<!--
 Copyright (C) 2018 Dzmitry Hatouka (Gotowka). htotatut@gmail.com

 This program is free software; you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation; either version 3, or (at your option)
 any later version.

 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.

 You should have received a copy of the GNU General Public License
 along with this program. See COPYING.txt
 If not, see <https://www.gnu.org/licenses/>.

 Please note that this program has CLA (Contributor Licence Agreement).
 You accept it by making any contribution to this program.
 See file CONTRIBUTE.txt for more details.
 List of 3-rd party components and their licenses is in 3RDPARTIES.txt
-->
<ui version="4.0">
 <class>Import</class>
 <widget class="QDialog" name="Import">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import</string>
  </property>
  <widget class="QGroupBox" name="gbSource">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Source</string>
   </property>
   <widget class="QRadioButton" name="rb2ormore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Gedmatch: 2 or more...</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbMatchingsegment">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Gedmatch: matching segment...</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbCSV">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>FTDNA or MyHeritage</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbDuplicates">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Duplicates by name, position, kit and chromosome</string>
   </property>
   <widget class="QRadioButton" name="rbIgnore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ignore</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbOverwrite">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Overwrite</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPreserve">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Preserve</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbMinimum">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum segment length: </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbSegment">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>18</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
